src/openfl/display/DisplayObject.hx

```diff
@@ -1670,6 +1671,21 @@ class DisplayObject extends EventDispatcher implements IBitmapDrawable #if openf
 		return value;
 		
 	}
+
+
+	private inline function __normalizeAngleAS3(value:Float):Float {
```

Comment:
the function can/should be static
also i think just `__normalizeAngle` is fine, the whole purpose of openfl is to behave like Flash anyway :)

Reply:
btw if you want to follow OpenFL code style, you need a space between function name and opening paren